.github/workflows/main.yml

```diff
@@ -26,7 +26,10 @@ jobs:
           version: 8.5.1
 
       - name: install packages
-        run: pnpm install
+        run: pnpm install --frozen-lockfile
```

Comment:
I guess that `--frozen-lockfile` is set to true by default in a CI environment, so we don't need to specify it?
https://pnpm.io/cli/install

Reply:
I see 👍 
It's new learning for me.
I fixed it!!

[abca3c3](https://github.com/poteboy/kuma-ui/pull/99/commits/abca3c3756308f1f42957bc496f8eda58aef4b01)